parlai/scripts/self_chat.py

```diff
@@ -103,7 +111,9 @@ def self_chat(opt, print_parser=None):
     if opt.get('display_examples'):
         print('-- end of episode --')
 
+    logger.reset_world()  # add to logs
```

Comment:
this is so confusing, how does this add to logs?!

Reply:
hmm yea not sure what this is.. was it not working before?.. oh like the last episode maybe wasn't flushed, or..?

Reply:
yeah it only flushes when the episode is done